### PR TITLE
fix: some nameprefix replacements from CARML migration needed to be changed to be AVM compliant

### PR DESCRIPTION
**Pull request by rahalan**
[[namePrefix]]
was replaced with
#_namePrefix_#

### CODE REVIEW

File: avm/res/web/site/README.md [hunk 1]
```diff
@@ -2118,7 +2118,7 @@ For all other app settings key-value pairs use this object.
 "appSettingsKeyValuePairs": {
     "value": {
       "AzureFunctionsJobHost__logging__logLevel__default": "Trace",
-      "EASYAUTH_SECRET": "https://adp-[[namePrefix]]-az-kv-x-001.vault.azure.net/secrets/Modules-Test-SP-Password",
+      "EASYAUTH_SECRET": "https://adp-#_namePrefix_#-az-kv-x-001.vault.azure.net/secrets/Modules-Test-SP-Password",
```
Comment (eriqua) [inline avm/res/web/site/README.md:2121]: @AlexanderSehr do we still have Parameter Usage content in READMEs?

Comment (AlexanderSehr) [reply avm/res/web/site/README.md:2121]: Hey @eriqua, only if it's part of the `## Notes`. @ahmadabdalla and I removed almost all and only kept a handful that we moved to the notes

Comment (rahalan) [reply avm/res/web/site/README.md:2121]: How to move forward? @eriqua @AlexanderSehr

Comment (eriqua) [reply avm/res/web/site/README.md:2121]: We should probably rediscuss this some time as this content is very likely to get stale. I'll approve the PR meanwhile as it won't harm

**Review by eriqua — APPROVED** [review]
🆗

File: avm/res/web/site/README.md (hunk 1 above)
Comment (AlexanderSehr) [reply avm/res/web/site/README.md:2121]: Happy to. Kept it only where it seemed meaningful

**Pull request merged**